src/lbfgs.jl

```diff
@@ -146,7 +152,7 @@ function solve!(
 
     # Perform improved Armijo linesearch.
     t, good_grad, ft, nbk, nbW =
-      armijo_wolfe(h, f, slope, ∇ft, τ₁ = T(0.9999), bk_max = 25, verbose = false)
+      armijo_wolfe(h, f, slope, ∇ft, τ₁ = τ₁, bk_max = bk_max, verbose = Bool(verbose_subsolver))
```

Comment:
Why is `verbose` and int converted to bool?

Reply:
I think it is to be consistent with `trunk` https://github.com/JuliaSmoothOptimizers/JSOSolvers.jl/blob/f91f34d4eb374323ea582838a2747d2ddeaa2a09/src/trunk.jl#L129